gcemetadata/gcemetadata

```diff
@@ -115,9 +115,7 @@ def main():
         getopt_metaopts.append('diskid=')
         getopt_metaopts.append('licenseid=')
         getopt_metaopts.append('netid=')
-        getopt_metaopts.append('output=')
         getopt_metaopts.append('query=')
```

Comment:
No they are in the general options list, which gets added, thus these were duplicate entries all along